modules/acquia_cms_common/tests/src/Functional/ContentTypeTestBase.php

```diff
@@ -271,9 +271,9 @@ protected function doTestEditorAccess() {
     $this->drupalGet('/node/1/delete');
     $assert_session->statusCodeEquals(200);
 
-    // Test that we can clone content.
+    // Test that we can not clone content.
     $this->drupalGet('/entity_clone/node/1');
-    $assert_session->statusCodeEquals(200);
+    $assert_session->statusCodeEquals(403);
```

Comment:
Remove the completed section no, need to check for 403 here.

Reply:
If we are not removing permission: https://github.com/acquia/acquia_cms/blob/ACMS-1606/modules/acquia_cms_common/src/Facade/PermissionFacade.php#L145 we can keep this test to check for 403

Reply:
Make sense, @deepakmishra2 please remove this assertion.